validation/connector_plugin_manifest_latest.xsd

```diff
@@ -172,4 +172,9 @@
   <xs:complexType name="Platform-ST">
     <xs:attribute name="name" type="OSShortName-ST" use="required"/>
   </xs:complexType>
+  <xs:simpleType name="CompanyName-ST">
```

Comment:
This is better named NonEmptyString-ST, since it might be used for other attributes later.

Reply:
Renamed. Thanks for the review!